doc/nrf/releases/release-notes-latest.rst

```diff
@@ -12,6 +12,12 @@ The most relevant changes that are present on the master branch of the |NCS|, as
 .. note::
     This file is a work in progress and might not cover all relevant changes.
 
+
+Highlights
+**********
+
+* Added HomeKit ADK repository available for verified company in MFi Program.
```

Comment:
```suggestion
* Added HomeKit ADK repository available for verified companies participating in the MFi Program.
```
Not sure if this should be part of the Highlights section, can be decided by @carlescufi or during the doc review meeting.